examples/tree_2d_dgsem/elixir_advection_diffusion_nonperiodic.jl

```diff
@@ -0,0 +1,87 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection-diffusion equation
+
+get_diffusivity() = 5.0e-2
+advection_velocity = (1.0, 0.0)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+# Note: If you change the diffusion parameter here, also change it in the initial condition
```

Comment:
```suggestion
```
I think `get_diffusivity()` takes care of this issue

Reply:
I agree. In this case, this note can be removed from a few other elixirs in this PR, too, I believe.